src/components/Settings/Tabs/VueTorrent/VGeneral.vue

```diff
@@ -242,6 +252,20 @@ export default {
     async fetchQbitVersion() {
       this.Qbitversion = await Qbit.getAppVersion()
     },
+    importSettings() {
+      try {
+        const userState = JSON.parse(this.settingsField)
+      } catch (e) {
+        alert('Invalid JSON!')
```

Comment:
Would probably be better to replace this with something like 
`Vue.$toast.error(....)`